README.md

```diff
@@ -322,13 +323,14 @@ sampling:
 
 > It is recommended that you use the latest [release](https://github.com/census-instrumentation/opencensus-service/releases).
 
-The collector can be run directly from sources, binary, or a Docker image.
+The collector can be run directly from sources, binary, or a Docker image. If you are planning to run from sources or build
+on your machine start by cloning the repo using `go get -d github.com/census-instrumentation/opencensus-service`.
 
 The minimum Go version required for this project is Go 1.11.4.
 
 1. Run from sources:
 ```shell
-$ GO111MODULE=on go run github.com/census-instrumentation/opencensus-service/cmd/occollector
+$ GO111MODULE=on go run github.com/census-instrumentation/opencensus-service/cmd/occollector --help
```

Comment:
Do you need the help?

Reply:
not necessarily but both agent and collector will need some parameters to run, so instead of running and getting the error message this already guides to what is needed